src/app/app.module.ts

```diff
@@ -4,16 +4,144 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { FormsModule } from '@angular/forms';
 import { NgxDatatableModule } from '@swimlane/ngx-datatable';
 
-import { AppComponent } from './app.component';
-import { CounterComponent } from './counter.component';
 import { MomentModule } from 'ngx-moment';
-import { NgxUIModule } from '../../projects/swimlane/ngx-ui/src/public_api';
+import { NgxUIModule, DateTimeComponent } from '../../projects/swimlane/ngx-ui/src/public_api';
 
 import { HttpClientModule } from '@angular/common/http';
 import { FileUploadModule } from 'ng2-file-upload';
 import { FlexLayoutModule } from '@angular/flex-layout';
+
+import { RouterModule, Routes } from '@angular/router';
+
+// Components
+import { AppComponent } from './app.component';
+import { CounterComponent } from './counter.component';
```

Comment:
The counter component is a helper component for the Tabs page, so let's move it into `/components/tabs-page`

Reply:
Don't you have to register `CounterComponent` in a module though?

Reply:
Yes, just commenting here for the file location.

Reply:
Okay I understand what you mean now